BeagleBone_black/src/communications/communications.hpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * Authors: Kofi and Isabella
+ * Organisation: HYPED
+ * Date: 1. April 2018
+ * Description:
+ *
+ *    Copyright 2018 HYPED
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#ifndef BEAGLEBONE_BLACK_COMMUNICATIONS_COMMUNICATIONS_HPP_
+#define BEAGLEBONE_BLACK_COMMUNICATIONS_COMMUNICATIONS_HPP_
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
```

Comment:
Use `cstdio`, `cstdlib` and `cstring`. Order the imports alphabetically. Separate sections by an empty line. See the styleguide.